docs/04-api-reference/front-commerce-core/key-values.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Key-Values"
+sidebar_position: 2
+description: "Manipulate object with key-values methods"
```

Comment:
**nitpick:** it's not very clear of when it is useful

**suggestion:** explain where developers might find it in our code and how it could solve their problems

```suggestion
description: "Manipulate object with key-values methods. This is convenient to convert free shaped objects to types compatible with GraphQL types."
```

Reply:
applied suggestion